.github/workflows/unit-test.yml

```diff
@@ -26,20 +26,22 @@ jobs:
         python-version: "3.8"
         cache: 'pip'
         cache-dependency-path: 'environments/**_requires.txt'
+    - name: Install dependencies
+      run: |
+        pip cache purge
+        python -m pip install --upgrade --no-cache-dir pip
+        pip install --no-cache-dir -v -e .[all]
+        pip cache purge
     - name: Increase swapfile
       run: |
         df -h
         free -h
         sudo swapoff -a
-        sudo fallocate -l 12G /swapfile
+        sudo fallocate -l 14G /swapfile
         sudo chmod 600 /swapfile
         sudo mkswap /swapfile
         sudo swapon /swapfile
         sudo swapon --show
```

Comment:
This file needs to be aligned with the version in PR #195 , which:
1. doesn't clean the pip caches. (Github Actions would cache pip dependencies, and it can save some time for later runs)
2. changes the swapfile directory from ` /swapfile` to `/mnt/swapfile`. (Save more spaces for `/`)

You can try to check if the version of this file in PR #195  works for this PR.

Reply:
have merged from PR #195.